pyfr/solvers/navstokes/kernels/bcs/sub-out-fp.mako

```diff
@@ -6,7 +6,7 @@
 % for i in range(nvars - 1):
     ur[${i}] = ul[${i}];
 % endfor
-    ur[${nvars - 1}] = ${c['p']/(c['gamma'] - 1)}
+    ur[${nvars - 1}] = (${c['p']})/${(c['gamma'] - 1)}
```

Comment:
I think after we merge the new space time Characteristic BC that we will not need the parenthesis.